include/deal.II/optimization/line_minimization.h

```diff
@@ -418,14 +420,14 @@ namespace LineMinimization
     const NumberType r1       = f2 - f1 - g1 * x2_shift;
     const NumberType r2       = f3 - f1 - g1 * x3_shift;
     const NumberType denom =
-      std::pow(x2_shift * x3_shift, 2) * (x2_shift - x3_shift);
+      Utilities::pow(2, x2_shift * x3_shift) * (x2_shift - x3_shift);
```

Comment:
Why did you switch the order of the arguments?  There are supposed to be same  as `std::pow`